missilewars-plugin/src/main/java/de/butzlabben/missilewars/util/stats/PlayerGuiFactory.java

```diff
@@ -72,7 +72,9 @@ public void openWhenReady(Player player) {
         int currentSize = names.size();
         if (realSize > currentSize) {
             if (Config.isContactAuth()) {
-                player.sendMessage(Messages.getPrefix() + "Fetching not cached player names: " + currentSize + "/" + realSize);
+                player.sendMessage(Messages.getMessage(true, Messages.MessageEnum.STATS_FETCHING_PLAYERS)
+                        .replace("currentSize", Integer.toString(currentSize))
```

Comment:
I think we should also stick here with the pattern for replacements and have it like:
"%current_size%"
"%real_size%"